show in red incorrect properties in settings.ini

lcaouen opened the pull request:
Shows in the About dialog box the incorrect settings from settings.ini.
It compares the settings from the settings.ini to the settings from the preferences.properties
The incorrect ones are displayed in red

This is related to this issue :
https://github.com/ControlSystemStudio/phoebus/issues/3605

- Testing:
    - tested manually by @lcaouen @minijackson 

- Documentation:
    - A message is displayed at the top of the view in the about dialog saying the settings in red are incorrects

georgweiss commented:
Unfortunately not able to get the intended behavior.
Starting without any path to a settings file I get:

<img width="789" height="642" alt="Screenshot 2025-12-01 at 09 16 20" src="https://github.com/user-attachments/assets/700758b3-77ae-477f-9bfd-07ede998e2f4" />

Starting with a custom settings file (correct settings) specified through ```-settings /path/to/settings/file.ini``` I get:

<img width="791" height="1085" alt="Screenshot 2025-12-01 at 09 39 03" src="https://github.com/user-attachments/assets/e8671ef2-8a05-4a85-9958-723beead00cc" />

lcaouen commented:
I have updated the source code.
If there is only one jar in the classpath, it opens it to get the paths of the other jars.
I have also added the "misssing code"
- a monospace font
- escape the html tags in the properties
- a message at the top saying properties in red are incorrects

<img width="671" height="105" alt="image" src="https://github.com/user-attachments/assets/77753eaf-adb5-4b38-9fee-38e21154053d" />

lcaouen commented:
Thanks.
I have just downloaded the files.
Also I have just pushed a new version which should work with IntelliJ and exclude some keys from the check.

Now I'm gonna have a look to the snap_widgets and alarm/server keys

lcaouen commented:
Do you use phoebus.sh(.bat) in your phoebus-product to run it ?
Do you add any argument like -settings to give your ini file ?

<img width="658" height="446" alt="image" src="https://github.com/user-attachments/assets/8189e902-dcd6-41d7-b554-0ec9cc5d9a80" />

When I run phoebus.bat without any argument (so no -settings given), I get this screen, which is correct as no settings file is loaded.
<img width="813" height="577" alt="image" src="https://github.com/user-attachments/assets/75ca70cd-aa1e-485f-b99a-580cf4efab58" />

Is that what you get or something different ?
